extensions/pkg/controller/dnsrecord/reconciler.go

```diff
@@ -235,3 +240,45 @@ func (r *reconciler) delete(ctx context.Context, dns *extensionsv1alpha1.DNSReco
 
 	return reconcile.Result{}, nil
 }
+
+func updateCreatedCondition(status extensionsv1alpha1.Status, conditionStatus gardencorev1beta1.ConditionStatus, reason, message string, updateIfExisting bool) error {
+	conditions := status.GetConditions()
+	c := gardencorev1beta1helper.GetCondition(conditions, extensionsv1alpha1.ConditionTypeCreated)
+	if c != nil && !updateIfExisting {
+		return nil
+	}
+	if c != nil && c.Status == conditionStatus {
+		return nil
+	}
+
+	builder, err := gardencorev1beta1helper.NewConditionBuilder(extensionsv1alpha1.ConditionTypeCreated)
+	if err != nil {
+		return err
+	}
+	if c != nil {
+		builder = builder.WithOldCondition(*c)
+	}
+
+	new, _ := builder.WithStatus(conditionStatus).WithReason(reason).WithMessage(message).Build()
+	status.SetConditions(gardencorev1beta1helper.MergeConditions(conditions, new))
+	return nil
+}
+
+func getCreatedConditionStatus(status extensionsv1alpha1.Status) gardencorev1beta1.ConditionStatus {
+	for _, c := range status.GetConditions() {
+		if c.Type == extensionsv1alpha1.ConditionTypeCreated {
+			return c.Status
+		}
+	}
+	return gardencorev1beta1.ConditionUnknown
+}
+
+func addCreatedConditionFalse(status extensionsv1alpha1.Status) error {
+	message := "error on initial record creation in infrastructure"
+	return updateCreatedCondition(status, gardencorev1beta1.ConditionFalse, "error", message, false)
```

Comment:
```suggestion
	message := "Error on initial record creation in infrastructure"
	return updateCreatedCondition(status, gardencorev1beta1.ConditionFalse, "Error", message, false)
```

Reply:
done

Reply:
Any reason to not capitalize the reason as suggested? Or was it by mistake?

Reply:
It was by mistake. But now it's done.